### DISCUSSION TITLE
Simple Code to Get ETFs from Sectors

### DISCUSSION THREAD

**Discussion by Dabs212** [JerBouma/FinanceDatabase#24] [Show and tell]
#pip install macrodatabase
#pin install --upgrade ta
#pip install financedatabase
#pip install thepassiveinvestor
#
import financedatabase as fd
import yfinance as yf
import pandas as pd
#from ta.volatility import BollingerBands
import matplotlib.pyplot as plt

equities_sectors = fd.show_options('equities', 'sectors')
sectors = equities_sectors[1:]
print(equities_sectors[1:])
#
for sector in sectors:
    try:
        etf1 = fd.select_etfs(category=sector)
        print(sector, ": ", etf1.keys())
    except ValueError:
        continue
#

**Comment by JerBouma** [comment]
Should now be even more efficient 🙃
